1_Diversity-Inclusion.md

```diff
@@ -1,29 +1,53 @@
-# Metric: Diversity and Inclusion
+# Category: Diversity and Inclusion
 
-Activity Metric | Description
+Goal: Identify the diversity and inclusion within a project. 
+
+## Individual Diversity
+
+Goal: Identify the diversity of individuals within a project.
+
+Name | Question
+--- | ---
+Contributor Demographics | What are the gender, age, location, education, and skills of contributors? 
+Leadership Demographics | What are the gender, age, location, education, and skills of leadership (e.g., Board, Technical Steering Committee, and Maintainers)?
+New Contributors* vs Maintainers** | What is the ratio of new contributors to maintainers?
+
+## Organizational Diversity
+
+Goal: Identify the diversity of organizations within a project.
+
+Name | Question
+--- | ---
+Contributing Organizations | What is the number of and who are the organizations contributing to the project?
+Contributions per Organization | What is the number of contributions from each organization?
+Contributors per Organization | What is the number of individual contributors from each organization?
+Maintainer Organizational Representation | What is the ratio and number of maintainers from each organization? 
+Maintainers per Organization | What is the ratio and number of maintainers from each organization? 
+Maintainer Commit Organizational Diversity | What is the number of commits a maintainer makes for contributors from each organization?

```

Comment:
What does this question get at? 

- What is, for each organization, the commit ratio for maintainers versus contributors from the same organization?